src/Schema/Values/ContextModule.ts

```diff
@@ -86,8 +86,14 @@ export enum ContextModule {
   marketInsights = "marketInsights",
   newWorksByArtistsYouFollowRail = "newWorksByArtistsYouFollowRail",
   newWorksByGalleriesYouFollowRail = "newWorksByGalleriesYouFollowRail",
-  ordersShipping = "ordersShipping",
```

Comment:
Can alphabetize this? (noted that it was out of alpha to begin with)

Reply:
ugh, I thought I did but apparently I don't know the English alphabet 😅 

👍 Will do.

Reply:
Resolved in most recent commit!